### ISSUE TITLE
Canproco - adding discs labels for session M12

### ISSUE THREAD

**Issue by NathanMolinier** [neuropoly/data-management#275]
## Description 

Discs labels for the session M12 were generated using the hourglass network however some manual corrections need to be done before merging.

The new branch is nm/add_discs-labels_ses-M12

**Comment by plbenveniste** [comment]
**QC of disc labels of M12 subjects**

The QC is available here : `duke.neuro.polymtl.ca/temp/plben/generate_qc_vert_levels_M12`. QC was obtained using `generate_qc.py` which is in the same folder. 

QC was performed over the 340 images. 

<details>
<summary> Identified 50 problematic subjects: </summary>

   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-tor035_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon032_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van182_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-tor061_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-tor092_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van177_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon005_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-edm118_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-edm127_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van124_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-cal104_ses-M12_STIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van201_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-tor141_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van208_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-cal134_ses-M12_STIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon185_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon149_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-cal194_ses-M12_STIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van213_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon198_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon138_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon199_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-cal143_ses-M12_STIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van155_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon072_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-edm161_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van198_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-edm113_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-tor098_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-tor053_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-edm114_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon096_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon065_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon009_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van175_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-tor090_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van174_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van189_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-cal165_ses-M12_STIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-edm005_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van055_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon172_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon121_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon132_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon168_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-cal176_ses-M12_STIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van216_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-mon160_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van194_ses-M12_PSIR.nii.gz
   - /Users/plbenveniste/Desktop/generate_qc_vert_levels_M12/sub-van135_ses-M12_PSIR.nii.gz
</details>

Disc label segmentation were manually corrected using the script : [manual_correction.py](https://github.com/spinalcordtoolbox/manual-correction/blob/main/manual_correction.py). 

The following file was removed : 
- sub-mon032_ses-M12_PSIR_label-discs.nii.gz (poor quality image: label disc file empty)

The following files were kept because some lesions are visible, even though the quality is not great :

- sub-mon005_ses-M12_PSIR_label-discs.nii.g : poor image quality
- sub-van201_ses-M12_PSIR_label-discs.nii.gz : poor image quality
- sub-tor141_ses-M12_PSIR_label-discs.nii.gz : poor image quality
- sub-cal134_ses-M12_STIR_label-discs.nii.gz : poor image quality
- sub-mon072_ses-M12_PSIR_label-discs.nii.gz : poor image quality
- sub-mon096_ses-M12_PSIR_label-discs.nii.gz : poor image quality
- sub-mon009_ses-M12_PSIR_label-discs.nii.gz : poor image quality

The corrections were pushed to the same branch and the file `sub-mon032_ses-M12_PSIR_label-discs.nii.gz` was deleted in the branch as well. 

Ready to review.

**Comment by mguaypaq** [comment]
- `git annex get derivatives/labels/` is happy, for all the new commits on the branch.
- bids-validator is irrelevant, since this PR only makes changes under `derivatives/`.
- I shortened the second "Author" field in the JSON sidecars, which contained redundant information.
- I standardized file names from `label-discs` to `labels-disc` (there were some of each).

Merged into `master`, and deleted the branch `nm/add_discs-labels_ses-M12`.
